### ISSUE TITLE
No way to choose what exe is used for BoxedWine with .zip archive

### ISSUE THREAD

**Issue by hello-smile6** [DustinBrett/daedalOS#170]
I want to try running Chromium, but BoxedWine chooses the wrong .exe file.

**Comment by DustinBrett** [comment]
Good point, that is indeed not ideal. I'll try and think of a simple solution. Possibly a custom shortcut could make this easier.
